api/Bluetooth.json

```
@@ -84,7 +84,8 @@
             "version_added": "6.0"
           },
           "webview_android": {
-            "version_added": false
+            "version_added": false,
+            "notes": "See <a href='https://crbug.com/1100993'>bug 1100993</a>"
```

Here and in the other changed lines:

```suggestion
            "notes": "See <a href='https://crbug.com/1100993'>bug 1100993</a>."
```

---

I've made the change everywhere and force pushed.